packages/storybook-vue/package.json

```diff
@@ -20,7 +20,9 @@
     "@storybook/vue3": "6.5.9",
     "@types/node": "16.11.43",
     "@utrecht/component-library-vue": "file:../component-library-vue",
+    "@utrecht/components": "file:../../components",
     "@utrecht/design-tokens": "file:../../proprietary/design-tokens",
+    "@utrecht/documentation": "file:../../documentation",
```

Comment:
Looks like `@utrecht/design-tokens` went missing in the refactoring.

Reply:
Because, it's already there https://github.com/nl-design-system/utrecht/commit/10912c30e4481643eae84727b528c7e5c76b2d49#diff-13e2799060dfc58b367662b1bb078d80f0cf4e488cfa7d3d94a976c37a45150bR24